journey-to-ontologize-metadata.md

```diff
@@ -1,5 +1,11 @@
 # Journey to ontologize metadata
 
+### Abstract
+
+TBD.
```

Comment:
Please add an abstract. Giovanni had suggested a summarized version of https://github.com/ord-premise/interoperability-guidelines/pull/2/files#diff-4963b6e28b248aa06755c02c1f767e5abfb83166f27d0a4666b1f94d71432c08L27-R33 as an abstract candidate.